test/router_decoder_custom_sup_SUITE.erl

```diff
@@ -0,0 +1,113 @@
+-module(router_decoder_custom_sup_SUITE).
+
+-export([
+    all/0,
+    init_per_testcase/2,
+    end_per_testcase/2
+]).
+
+-export([
+    v8_recovery_test/1
+]).
+
+-include_lib("common_test/include/ct.hrl").
+-include_lib("eunit/include/eunit.hrl").
+
+%%--------------------------------------------------------------------
+%% COMMON TEST CALLBACK FUNCTIONS
+%%--------------------------------------------------------------------
+
+%%--------------------------------------------------------------------
+%% @public
+%% @doc
+%%   Running tests for this suite
+%% @end
+%%--------------------------------------------------------------------
+all() ->
+    [
+        v8_recovery_test
+    ].
+
+%%--------------------------------------------------------------------
+%% TEST CASE SETUP
+%%--------------------------------------------------------------------
+init_per_testcase(TestCase, Config) ->
```

Comment:
I might not make a huge difference for 1 test, but we might be able to forego `init_per_testcase` and directly startup the `router_decoder_custom_sup` supervisor, since we don't touch anything else in the system.

Reply:
I omitted `init_per_testcase()` in an earlier iteration between commits. There was a lot of boilerplate code such as starting ETS, lager, etc., so reusing the existing pattern seemed more clean in the end.  (That earlier attempt is why there was the `ifdef(TEST)` wrapped around `delete()` that has since been removed.

Also, there will likely be another test added now for more nuanced error reporting back to the console, so the human doesn't just see the `ignoring_invalid_javascript` atom.

Reply:
Managed to work the additional test case into an existing test; see commit 772469b